libpod/networking_linux.go

```diff
@@ -355,6 +351,11 @@ func (r *Runtime) setupRootlessPortMapping(ctr *Container, netnsPath string) (er
 	cmd := exec.Command(fmt.Sprintf("/proc/%d/exe", os.Getpid()))
 	cmd.Args = []string{rootlessport.ReexecKey}
 	// Leak one end of the pipe in rootlessport process, the other will be sent to conmon
+
+	if ctr.rootlessPortSyncR != nil {
+		defer errorhandling.CloseQuiet(ctr.rootlessPortSyncR)
+	}
+
```

Comment:
Do you need a similar for rootlessSlirpSyncR?

Reply:
do you mean `rootlessSlirpSyncW`?  No, it is already closed in `libpod/oci_conmon_linux.go`